dali/test/python/nose_utils.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nose.tools as tools
+import re
+import fnmatch
+
+
+def assert_raises(exception, *args, **kwargs):
+
+    """
+    Wrapper combining `nose.tools.assert_raises` and `nose.tools.assert_raises_regex`.
+    Specify ``regex=pattern`` or ``glob=pattern`` to check error message of expected exception against the pattern.
+    Value for `glob` must be a string, `regex` can be either a literal or compiled regex pattern.
+    By default, the check will ignore case, if called with `glob` or a literal for `regex`.
+    To enforce case sensitive check pass ``match_case=True``.
+    Don't specify `match_case` if passing already compiled regex pattern.
+    """
+
+    glob = kwargs.pop("glob", None)
+    regex = kwargs.pop("regex", None)
```

Comment:
Again - why not named arguments in the function signature?
```Python
def assert_raises(exception, *args, regex = None, glob = None, **kwargs)```

Reply:
Sorry, I misunderstood what you meant.